runtime/lib/ttnn/operations/eltwise/binary.cpp

```diff
@@ -126,6 +127,10 @@ void run(const ::tt::target::ttnn::EltwiseOp *op, ProgramContext &context) {
     runEltwiseBinaryCompositeOP(op, tensorPool, ::ttnn::minimum);
     break;
   }
+  case ::tt::target::ttnn::EltwiseOpType::Remainder: {
+    runEltwiseBinaryCompositeOP(op, tensorPool, ::ttnn::remainder);
```

Comment:
After rebase please add this to binary_composite.cpp, otherwise looks good.

Reply:
Done.